src/client/componentsMealsharing/AddMeal.js

```diff
@@ -0,0 +1,142 @@
+import "./Meal.css";
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+
+export const AddMeal = () => {
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+  const [maxreservation, setMaxreservation] = useState("");
+  const [price, setPrice] = useState("");
+  const [created, setCreated] = useState([]);
+  const [meals, setMeals] = useState([]);
+  const [succeed, setSucceed] = useState("");
+
+  useEffect(() => {
+    fetchDataResult();
+  }, []);
+
+  const fetchDataResult = async () => {
+    const response = await fetch("http://localhost:3000/api/meals");
+    const Data = await response.json();
+    setMeals(Data);
```

Comment:
Should use camelCase for variable names :) 
```suggestion
    const data = await response.json();
    setMeals(data);
```

Reply:
Thanks for telling. right 👍